src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionsinglefilter/Example1.java

```diff
@@ -15,6 +15,9 @@
   </module>
 </module>
 */
-
+package com.puppycrawl.tools.checkstyle.filters.suppressionsinglefilter;
 // xdoc section -- start
+public class Example1 {
+}
```

Comment:
We need real example of code that supposed to be violated, but filter helps to avoid violation.
In this example config targeting MagicNumberCheck and other check. So please create java code in Example1 that has violations on such checks without filters, but when filter applied, there is no violation.

Reply:
For all 10. Users love you see examples. 
You will be pleased to see examples in few years, as you will forget how it works.